src/datahike/connector.cljc

```diff
@@ -34,7 +34,8 @@
   (let [{:keys [eavt aevt avet temporal-eavt temporal-aevt temporal-avet
                 schema rschema system-entities ident-ref-map ref-ident-map config
                 max-tx max-eid op-count hash meta store]} db
-        backend (di/konserve-backend (:index config) store)]
+        backend (di/konserve-backend (:index config) store)
+        in-memory? (not= :mem (-> config :store :backend))]
```

Comment:
`in-memory?` is exactly the opposite from what is being tested here, so this really has to be renamed

Reply:
Thanks, that was indeed precision in the wrong direction :smile: .